templates/chatbots/single_chatbot_home.html

```diff
@@ -219,11 +219,32 @@ <h3 class="font-bold text-lg">Link with {{ platform.label }}</h3>
 
     {% if perms.chat.view_chat %}
       <div role="tabpanel" class="tab-content" id="content-allsessions">
-        {% include "chatbots/components/session_table_actions.html" %}
-        <div id="sessions-table" data-url="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}">
+        <div class="relative">
+          {% include "chatbots/components/session_table_actions.html" %}
+          <div class="absolute top-0 right-0 flex items-center gap-2">
+            <button class="btn btn-ghost btn-sm"
+                    hx-get="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}"
+                    hx-target="#sessions-table"
+                    hx-swap="innerHTML"
+                    hx-indicator="#refresh-loader"
+                    title="Refresh sessions">
+              <i class="fa-solid fa-rotate"></i>
+            </button>
+            <span class="loading loading-spinner loading-sm htmx-indicator" id="refresh-loader"></span>
```

Comment:
use `htmx-show` so that the spinner isn't included in the layout:
```suggestion
            <span class="loading loading-spinner loading-sm htmx-show" id="refresh-loader"></span>
```

(is `loading-sm` right or should it be `loading-xs`?)

Reply:
we are using sm at most places in the project. currently the size of loader is same as of refresh icon. we can use xs if we want smaller than it.